.goreleaser.yml

```
@@ -29,17 +29,19 @@ changelog:
     exclude:
     - '^docs:'
     - '^test:'
+
 archives:
-  -
-    name_template: '{{ .ProjectName }}_{{.Env.RELEASE_VERSION}}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
-    replacements:
-      darwin: darwin
-      linux: linux
-      386: 386
-      amd64: amd64
-    format_overrides:
-    - goos: windows
-      format: zip
+  - id: archives
+    name_template: >-
+      {{- .ProjectName }}_
+      {{.Env.RELEASE_VERSION}}_
```

It's probably missing `-` operator to strip leading spaces?
```suggestion
      {{- .Env.RELEASE_VERSION }}_
```